wforce/twmap-wrapper.cc

```diff
@@ -100,11 +120,11 @@ std::string TWStringStatsDBWrapper::getStringKey(const TWKeyType vkey)
   else if (vkey.which() == 2) {
     const ComboAddress ca =  boost::get<ComboAddress>(vkey);
     if (ca.isIpv4()) {
-      uint8_t v4_prefix = sdbp->getv4Prefix();
+      uint8_t v4_prefix = sdbvp->at(0)->getv4Prefix();
```

Comment:
Don't you want to use the computed shard index ?

Reply:
Well all the shards will have the same prefix so it doesn't really matter